src/domain/Progress/ProgressTracker/ProgressTracker.examples.md

```diff
@@ -20,19 +26,24 @@ initialState = { value: 1 };
     <ProgressTracker.Step label='Add Cost and  Follow Up' onClick={() => setState({value:2})}/>
 </ProgressTracker>
 ```
-### Steps
-Progress Tracker require at least two steps, with no more than six.
-
-<br />
 
 ### Best Practices
+* Progress Tracker should only be used for tasks requiring 2 to 6 steps.
 * If a task needs more than 6 steps, consider simplifying the process or breaking it up into multiple tasks.
 Use labels that clearly indicate the purpose of the step.
 * Keep labels to a single line of text, be short and concise (1-2 words).
 
 <br />
 
 ### Design Pattern
-* When progress tracker have less steps (~2 steps), the progress tracker should place 50% content to avoid the steps stay far away. 
+* When progress tracker have less steps (~2 steps), the progress tracker would place 50% content to avoid the steps stay far away. 
```

Comment:
`ProgressTracker will handle its own width based on the number of steps involved. Always ensure that the ProgressTracker is not in any surrounding boxes so that it will always take up 100% of the page content width`

Reply:
Maybe better to move this under Best Practices too.